frontend/src/pages/PipelineDetails.tsx

```diff
@@ -587,6 +586,8 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
       this.props.history.replace({
         pathname: `/pipelines/details/${this.state.v1Pipeline.id}/version/${versionId}`,
       });
+      this.props.updateToolbar(this.getInitialToolbarState());
+      this.props.updateToolbar({ pageTitle });
```

Comment:
Thank you Joe for the change, can you alternate the `pageTitle` before calling `updateToolbar`?

That is:
```
let toolbarState = this.getInitialToolbarState();
toolbarState.pageTitle = pageTitle;
this.props.updateToolbar(toolbarState);
```

In this case, we avoid the intermediate state in toolbar rendering.